src/TEC/Plugin.php

```diff
@@ -221,6 +221,10 @@ function maybe_update_post( int $post_id, $xml_node, bool $is_update ): void {
 		$this->container->make( Post_Handler::class )->maybe_update_post( $post_id, $xml_node, $is_update );
 	}
 
+	public function relink_posts_to_series( $value, $post_id, $key, $original_value, $existing_meta_keys, $import_id ) {
```

Comment:
Missing doc-block

Reply:
On the front of readability: if `value` and `key` are meta value and meta key, why not call them `meta_value` and `meta_key`?

Reply:
I was using the defaults from the WPAI docs. Makes sense, I adjusted.